addons/auth_totp/views/user_preferences.xml

```diff
@@ -1,51 +1,64 @@
+                    <group>
+                        <div>
+                            <div class="o_horizontal_separator d-flex align-items-center mt-0">2-Step Authentication
+                                <div attrs="{'invisible': [('totp_enabled', '!=', False)]}">
+                                    <button attrs="{'invisible': &quot;[('id', '=', uid)]&quot;}" name="totp_enable_wizard"
+                                            disabled="1" type="object" class="fa fa-toggle-on o_auth_2fa_btn disabled" aria-label="Enable TOTP"></button>
+                                    <button attrs="{'invisible': &quot;[('id', '!=', uid)]&quot;}" name="totp_enable_wizard"
+                                        type="object" class="fa fa-toggle-on o_auth_2fa_btn disabled" aria-label="Enable TOTP"></button>
+                                    <button groups="base.group_erp_manager" attrs="{'invisible': &quot;[('id', '=', uid)]&quot;}"
+                                        name="totp_invite" string="Invite to use TOTP" type="object" class="btn btn-primary"></button>
+                                </div>
+                                <button attrs="{'invisible': [('totp_enabled', '=', False)]}" name="totp_disable" type="object"
+                                        class="fa fa-toggle-on o_auth_2fa_btn enabled" aria-label="Disable TOTP"></button>
+
+                            </div>
+                            <span attrs="{'invisible': [('totp_enabled', '!=', False)]}" class="text-muted">
+                                2-Step authentication ("2FA") is a system of double authentication. The first one is your usual password and the second one is a code you get from a phone app or a sms. Odoo advises you to install a 2FA phone-based authenticator like Authy on your phone.
+                            <a href="https://www.odoo.com/documentation/14.0/applications/general/auth/2fa.html"
+                               title="Learn More" target="_blank">Learn More</a>
                         </span>
-                    </div>
-                </group>
+                        </div>
+                    </group>
+                </page>
             </xpath>
         </field>
     </record>
 
     <record model="ir.actions.server" id="action_disable_totp">
-        <field name="name">Disable TOTP on users</field>
+        <field name="name">Disable 2-Step authentication</field>
         <field name="model_id" ref="base.model_res_users"/>
         <field name="binding_model_id" ref="base.model_res_users"/>
         <field name="binding_view_types">list</field>
         <field name="state">code</field>
         <field name="code">
             action = records.totp_disable()
         </field>
-        <field name="groups_id" eval="[(4, ref('base.group_erp_manager'), 0)]"/>
+        <field name="groups_id" eval="[(4, ref('base.group_erp_manager'))]"/>
     </record>
 
     <record model="ir.ui.view" id="view_totp_wizard">
```

Comment:
Side note: could be great to move this view into wizard/auth_totp_wizard_views.xml, in order to cleanup a bit the mess in this file... and move the python code in wizard also, currently located in res_users.py because "who know why". In short, a [MOV] commit to make Tde happy :D .